src/main/java/edu/cornell/kfs/tax/batch/CUTaxBatchConstants.java

```diff
@@ -128,6 +131,10 @@ private CommonVendorFieldNames() {
         public static final String VENDOR_OWNERSHIP_CODE = CommonVendorFieldNames.VENDOR_OWNERSHIP_CODE;
         public static final String VENDOR_OWNERSHIP_CATEGORY_CODE = CommonVendorFieldNames.VENDOR_OWNERSHIP_CATEGORY_CODE;
         public static final String VENDOR_FOREIGN_IND = CommonVendorFieldNames.VENDOR_FOREIGN_IND;
+        //fields from AP_PMT_RQST_T (PaymentRequestDocument)
+        public static final String TAX_CLASSIFICATION_CODE = PurapPropertyConstants.TAX_CLASSIFICATION_CODE;
+        //fields from FP_DV_DOC_T
+        public static final String DV_PAYEE_TAX_CONTROL_CODE = KFSPropertyConstants.DISB_VCHR_PAYEE_TAX_CONTROL_CODE;

```

Comment:
If this constant is not being used, then I would suggest removing it if it's not actually needed.